app/views/notifiers/course/video_notifier/opening/course_notifications/email.html.slim

```diff
@@ -0,0 +1,10 @@
+- video = @object
+- course = video.course
+- host = course.instance.host
+
+- message.subject = t('.subject', course: course.title, video: video.title)
+
+/ TODO: Re-enable when routes are updated
```

Comment:
Not about the pending tests but that `course_video_path` doesn't exist yet, so I can't add it into the notifier yet.